### ISSUE TITLE
It doesn´t wanna import @angular/material": "^8.2.3"

### ISSUE THREAD

**Issue by saessugo** [unlight/sublime-import-helper#79]
![image](https://user-images.githubusercontent.com/29466915/80322503-b44d7800-87f3-11ea-8a9d-23b115e74b85.png)

It's the only problem I have, all the other modules are imported well. Can't seem to figure out, I thought maybe was a permission issue but I already gave that folder all the permissions for all the users, just in case, I'm on Windows 10 btw

**Comment by unlight** [comment]
`@angular/material` has a lot of modules and submodules.
Error happened from `esm-exports` module, looks like search of imports gone to infinite recursive loop.
```
load_module_timer: loading_modules: {'@angular/material': 0}
load_module_timer: loading_modules: {'@angular/material': 1}
load_module_timer: loading_modules: {'@angular/material': 2}
load_module_timer: loading_modules: {'@angular/material': 3}
load_module_timer: loading_modules: {'@angular/material': 4}
load_module_timer: loading_modules: {'@angular/material': 5}
load_module_timer: loading_modules: {'@angular/material': 6}
load_module_timer: loading_modules: {'@angular/material': 7}
load_module_timer: loading_modules: {'@angular/material': 8}
load_module_timer: loading_modules: {'@angular/material': 9}
load_module_timer: loading_modules: {'@angular/material': 10}
load_module_timer: loading_modules: {'@angular/material': 11}
load_module_timer: loading_modules: {'@angular/material': 12}
load_module_timer: loading_modules: {'@angular/material': 13}
load_module_timer: loading_modules: {'@angular/material': 14}
load_module_timer: loading_modules: {'@angular/material': 15}
Exec error: RangeError: Maximum call stack size exceeded
    at \Progs\Sublime-Text-3\Data\Packages\ImportHelper\node_modules\esm-exports\index.js:74:28
    at Generator.next (<anonymous>)
    at fulfilled (\Progs\Sublime-Text-3\Data\Packages\ImportHelper\node_modules\tslib\tslib.js:107:62)
```

I could not find effective algorithm to extract imports from module

`esm-exports` requires heavy refactoring

**Comment by unlight** [comment]
@19gonchi92 Should be fixed in version `3.0.0`, please check.
